packages/twenty-front/src/pages/settings/developers/webhooks/SettingsDevelopersWebhookDetail.tsx

```diff
@@ -55,6 +55,18 @@ export const SettingsDevelopersWebhooksDetail = () => {
                 fullWidth
               />
             </Section>
+            <Section>
+              <H2Title
+                title="Description"
+                description="An optional description"
+              />
+              <TextInput
+                placeholder="Description"
+                value={webhookData.description}
+                disabled
```

Comment:
should be enabled

Reply:
When edited, save button and cancel button should be displayed so we can save the change. You can see how this is done for `SettingsObjectEdit` component